finite_state_machine/state_machine.py

```diff
@@ -21,6 +40,37 @@ class Transition(NamedTuple):
     on_error: Union[bool, int, str]
 
 
+class TransitionMeta(object):
+    def __init__(self, name):
+        self.name = name
+        self.transitions = {}
+
+    def get_transition(self, source):
+        transition = self.transitions.get(source, None)
+        if transition is None:
+            transition = self.transitions.get("*", None)
+        if transition is None:
+            transition = self.transitions.get("+", None)
+        return transition
```

Comment:
The `+` and `*` functionality has not been added to the library. Would need to add tests to ensure it works as you expect.

TBH, I'm not sure I want it at this stage. This could be worth creating another issue (with separate PR), but until the need arises, I think it's pragmatic to leave it out.

Reply:
I will remove this.